src/main/java/seedu/address/logic/commands/CountryNoteViewCommand.java

```diff
@@ -33,11 +33,23 @@ public CountryNoteViewCommand(Country country) {
         this.country = country;
     }
 
+    /**
+     * Initializes a CountryNoteViewCommand that views all country notes.
+     */
+    public CountryNoteViewCommand() {
+        this.country = Country.getNullCountry();
+    }
+
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredCountryNoteList(countryNote -> countryNote.getCountry().equals(country));
-        return new CommandResult(String.format(MESSAGE_SUCCESS, country), false, false, WidgetViewOption.COUNTRY_NOTES);
+        if (country == null) {
```

Comment:
When will this be the case? Should it be country == Country.getNullCountry()?

Reply:
Yea, I change to NULL_COUNTRY but forgot to update null checks.